anthropic-java-bedrock/src/main/kotlin/com/anthropic/bedrock/backends/BedrockBackendAdapter.kt

```diff
@@ -0,0 +1,592 @@
+         */
+        private const val SERVICE_NAME = "bedrock-runtime"
+
+        /**
+         * The Anthropic version for the Bedrock service.
+         */
+        private const val ANTHROPIC_VERSION = "bedrock-2023-05-31"
+
+        /**
+         * The name of the header that identifies the Anthropic beta versions.
+         * To specify multiple beta versions, there can be more than one header,
+         * or the value of a header can be a comma-separated list of beta
+         * versions.
+         */
+        // For HTTP/2 responses, the header names are expected to be lower case.
+        private const val HEADER_ANTHROPIC_BETA = "anthropic-beta"
+
+        /**
+         * The name of the header that identifies the content type of the data
+         * in the response.
+         */
+        private const val HEADER_CONTENT_TYPE = "content-type"
+
+        /**
+         * The name of the header that identifies the content type for the
+         * "payloads" of AWS _EventStream_ messages in streaming responses from
+         * Bedrock.
+         */
+        private const val HEADER_PAYLOAD_CONTENT_TYPE =
+            "x-amzn-bedrock-content-type"
+
+        /**
+         * The content type for Bedrock responses containing data in the AWS
+         * _EventStream_ format. The value of the [HEADER_PAYLOAD_CONTENT_TYPE]
+         * header identifies the content type of the "payloads" in this stream.
+         */
+        private const val CONTENT_TYPE_AWS_EVENT_STREAM =
+            "application/vnd.amazon.eventstream"
+
+        /**
+         * The content type for Anthropic responses containing Bedrock data
+         * after it has been translated into the Server-Sent Events (SSE) stream
+         * format.
+         */
+        private const val CONTENT_TYPE_SSE_STREAM =
+            "text/event-stream; charset=utf-8"
+
+        /**
+         * The content type for data in JSON format. The bodies of all requests
+         * are expected to have this content type. For responses using the AWS
+         * _EventStream_ content type, the "payloads" of the event messages are
+         * expected to have this content type.
+         */
+        private const val CONTENT_TYPE_JSON = "application/json"
+
+        /**
+         * Creates a new builder for configuring and creating a new instance of
+         * [BedrockBackendAdapter].
+         */
+        @JvmStatic fun builder() = Builder()
+
+        /**
+         * Creates new [BedrockBackendAdapter] with credentials resolved from
+         * the environment or configuration files.
+         *
+         * This is a convenience method that is the equivalent of calling:
+         *
+         * ```
+         * BedrockBackendAdapter.builder().fromEnv().build()
+         * ```
+         *
+         * @throws AnthropicException See [Builder.build] for details.
+         */
+        @JvmStatic fun fromEnv(): BedrockBackendAdapter {
+            return builder().fromEnv().build()
+        }
```

Comment:
Throughout the PR, would be nice to make these "lone return" cases just use the Kotlin shorthand

In this case:
```kotlin
 @JvmStatic fun fromEnv(): BedrockBackendAdapter = builder().fromEnv().build()
```